### PR TITLE
Add root values to walkStackFrames entry tracepoint

### PR DESCRIPTION
**Pull request by gacholio**
Also add the tracepoints to the normal (non-verbose) walker.

Signed-off-by: Graham Chapman <graham_chapman@ca.ibm.com>

### CODE REVIEW

**Comment by gacholio** [comment]
@keithc-ca Is this the appropriate way to add parameters to a tracepoint?

File: runtime/verbose/j9vrb.tdf [hunk 1]
```diff
@@ -23,7 +23,7 @@ Submodules=j9vmutil,j9util,pool,avl
 DATFileName=J9TraceFormat.dat
 
 TraceEvent=Trc_VRB_VMInitStages_Event1 Overhead=1 Level=1 Template="Trace engine initialized for module j9vrb"
-TraceEntry=Trc_VRB_WalkStackFrames_Entry Overhead=1 Level=1 Template="WalkStackFrames - walkThread=%p flags=%p"
+TraceEntry=Trc_VRB_WalkStackFrames_Entry_OBSOLETE Obsolete Overhead=1 Level=1 Template="WalkStackFrames - walkThread=%p flags=%p"
```
[the remark this answers is not in the document]
Comment (gacholio) [reply runtime/verbose/j9vrb.tdf]: Undone.

**Comment by keithc-ca** [comment]
> @keithc-ca Is this the appropriate way to add parameters to a tracepoint?

Yes, other than the needless name change to the existing tracepoint.

**Comment by gacholio** [comment]
jenkins test sanity win jdk8

**Comment by gacholio** [comment]
jenkins test sanity win jdk8

**Comment by gacholio** [comment]
jenkins test sanity win jdk8

**Comment by keithc-ca** [comment]
Jenkins test sanity osx jdk11

**Review by tajila — APPROVED** [review]

**Comment by keithc-ca** [comment]
I notice that `swalk.c` doesn't compile if `J9VM_INTERP_STACKWALK_TRACING` is defined and I don't see any way to configure a build so it would be: perhaps a follow-on change should remove the code for that case.

**Comment by gacholio** [comment]
That define should be on in the verbose library where the file is recompiled.

**Comment by keithc-ca** [comment]
> That define should be on in the verbose library where the file is recompiled.

You're right, I missed that; and it does compile cleanly (from there).

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**